### PR TITLE
Added memory recommendation

### PR DESCRIPTION
**Pull request by tom-taylor**

### CODE REVIEW

File: source/operatingsystems/linux/elasticsearch/elasticsearch.md [hunk 1]
```diff
@@ -102,6 +102,19 @@ network.host: 10.0.0.17
 
 The Elasticsearch service will need a restart after this change.
 
+### Memory Limit/Heap Size
+On recent versions of Elasticsearch, the default memory allocation is 32GB which can cause issues. We recommend that you set to 2GB in `/etc/elasticsearch/jvm.options`
```
Comment (terencesimms) [inline source/operatingsystems/linux/elasticsearch/elasticsearch.md]: Might be worth elaborating on what issues would be seen, however brief.

File: source/operatingsystems/linux/elasticsearch/elasticsearch.md [hunk 2]
```diff
@@ -102,6 +102,19 @@ network.host: 10.0.0.17
...
+
+```bash
+vim /etc/elasticsearch/jvm.options
+
+ -Xms2g
+ -Xmx2g
+
+```
+
+The Elasticsearch service will need a restart after this change.
```
Comment (terencesimms) [inline source/operatingsystems/linux/elasticsearch/elasticsearch.md:116]: Worth including the restart command within this, if referencing it.

**Review by terencesimms — DISMISSED** [review]
just two small changes to be made

File: source/operatingsystems/linux/elasticsearch/elasticsearch.md (hunk 2 above)
Comment (tom-taylor) [reply source/operatingsystems/linux/elasticsearch/elasticsearch.md:116]: Changed them now

File: source/operatingsystems/linux/elasticsearch/elasticsearch.md [hunk 3]
```diff
@@ -102,6 +102,25 @@ network.host: 10.0.0.17
 
 The Elasticsearch service will need a restart after this change.
 
+### Memory Limit/Heap Size
+As of ElasticSearch version 7, the default memory allocation is no longer 2GB. The service will make assumptions based on total memory and can potentially set the heap size as high as 32GB, which can cause "Out of Memory" issues if left unchanged. We recommend that you set this to 2GB in `/etc/elasticsearch/jvm.options`
```
Comment (miff2000) [inline source/operatingsystems/linux/elasticsearch/elasticsearch.md]: This is failing CI for the case used here. It should be `Elasticsearch`. I'll fix, but for your own awareness :)

**Review by miff2000 — APPROVED** [review]

**Comment by miff2000** [comment]
Merged. Cheers for your contribution @tom-taylor

**Pull request merged**